tests/phpt/handler_should_report_user_errors.phpt

```diff
@@ -12,6 +12,12 @@ trigger_error('abc', E_USER_ERROR);
 
 var_dump('I should not be reached');
 ?>
+--SKIPIF--
+<?php
+if (PHP_VERSION_ID >= 80400) {
+    echo 'SKIP — this test uses methods depreciated in PHP 8.4+';
```

Comment:
I think it would be worth noting this on the PR description – i.e. you can't trigger errors with `E_USER_ERROR`

```suggestion
    echo 'SKIP — this test uses methods deprecated in PHP 8.4+';
```